add xxd via vim as a dep for ubuntu?

Put ```apt-get install vim``` in ubuntu_sim_nuttx.sh

like with  PR #301 

I was building on ~~docker~~ **a stock ubuntu:16.04 docker image that I set up with  ubuntu_sim_nuttx.sh** and got error that xxd was not installed:

```bash
make px4fmu-v5_default
-- PX4 VERSION: v1.8.0-1490-gf637f6a
# BLAH BLAH BAH ...
-- ROMFS: px4fmu_common
CMake Error at ROMFS/CMakeLists.txt:157 (message):
  xxd not found
```

as the msg implies, xxd is used to do something in ROMFS/CMakeLists.txt

```CMake
# create nsh_romfsimg.c
find_program(XXD xxd)
if(NOT XXD)
    message(FATAL_ERROR "xxd not found")
endif()
add_custom_command(OUTPUT nsh_romfsimg.c
        COMMAND ${CMAKE_COMMAND} -E remove -f nsh_romfsimg.c
        COMMAND ${XXD} -i romfs.img nsh_romfsimg.c
        COMMAND sed 's/unsigned/const unsigned/g' nsh_romfsimg.c > nsh_romfsimg.c.tmp && ${CMAKE_COMMAND} -E rename nsh_romfsimg.c.tmp nsh_romfsimg.c
        DEPENDS romfs.img
        )
```

So after a google search, I find out it is distributed with apt in ```vim-common```.

I installed vim (```apt install vim```) and now it compiles.


Another solution would be to switch to something more common like ```od```?

Anyway, maybe this issue (/ pr if someone makes one) might help someone someday

--- a/build_scripts/ubuntu_sim_common_deps.sh
+++ b/build_scripts/ubuntu_sim_common_deps.sh
@@ -20,7 +20,7 @@ sudo apt-get remove modemmanager -y
 # Common dependencies
 echo "Installing common dependencies"
 sudo apt-get update -y
-sudo apt-get install git zip qtcreator cmake build-essential genromfs ninja-build exiftool -y
+sudo apt-get install git zip qtcreator cmake build-essential genromfs ninja-build exiftool vim-common -y
 # Required python packages
 sudo apt-get install python-argparse python-empy python-toml python-numpy python-dev python-pip -y
 sudo -H pip install --upgrade pip

--- a/build_scripts/windows_bash_nuttx.sh
+++ b/build_scripts/windows_bash_nuttx.sh
@@ -37,7 +37,7 @@ fi
 echo "Installing common dependencies"
 sudo add-apt-repository ppa:george-edison55/cmake-3.x -y
 sudo apt-get update
-sudo apt-get install python-argparse git git-core wget zip python-empy python-toml python-numpy cmake build-essential genromfs -y
+sudo apt-get install python-argparse git git-core wget zip python-empy python-toml python-numpy cmake build-essential genromfs vim-common -y
 # required python packages
 sudo apt-get install python-dev -y
 sudo apt-get install python-pip -y

--- a/en/setup/dev_env_linux_centos.md
+++ b/en/setup/dev_env_linux_centos.md
@@ -20,7 +20,7 @@ easy_install pexpect
 easy_install toml
 easy_install pyyaml
 easy_install cerberus
-yum install openocd libftdi-devel libftdi-python python-argparse flex bison-devel ncurses-devel ncurses-libs autoconf texinfo libtool zlib-devel cmake
+yum install openocd libftdi-devel libftdi-python python-argparse flex bison-devel ncurses-devel ncurses-libs autoconf texinfo libtool zlib-devel cmake 
 ```
 
 > **Note** You may want to also install  python-pip and screen

--- a/en/setup/dev_env_linux_centos.md
+++ b/en/setup/dev_env_linux_centos.md
@@ -20,7 +20,7 @@ easy_install pexpect
 easy_install toml
 easy_install pyyaml
 easy_install cerberus
-yum install openocd libftdi-devel libftdi-python python-argparse flex bison-devel ncurses-devel ncurses-libs autoconf texinfo libtool zlib-devel cmake 
+yum install openocd libftdi-devel libftdi-python python-argparse flex bison-devel ncurses-devel ncurses-libs autoconf texinfo libtool zlib-devel cmake vim-common
 ```
 
 > **Note** You may want to also install  python-pip and screen

--- a/en/setup/dev_env_linux_ubuntu.md
+++ b/en/setup/dev_env_linux_ubuntu.md
@@ -76,7 +76,7 @@ Update the package list and install the following dependencies for all PX4 build
 ```sh
 sudo apt-get update -y
 sudo apt-get install git zip qtcreator cmake \
-    build-essential genromfs ninja-build exiftool -y
+    build-essential genromfs ninja-build exiftool vim-common -y
 # Required python packages
 sudo apt-get install python-argparse \
     python-empy python-toml python-numpy python-yaml \

